docs/developer/general/policy_based_access_controls.rst

```diff
@@ -19,50 +20,51 @@ In the recommended architecture of the PBAC access control paradigm, the
 standard workflow is as follows:
 
 - The *Policy Enforcement Point* (PEP) is the client-facing service. It is
-  responsible for guarding the resource. When the PEP receives a request to access
-  a resource (1) it   inspects the request and combines it with other contextual
-  information to generate a authorization request to the *Policy Decision Point*
-  (PDP) (2). The PEP enforces the permissions granted   by the PDP and calls the
-  underlying API business logic to the protected resource backend only if the PDP
-  allows it (8,9,10).
+  responsible for guarding the resource. When the PEP receives a request to
```

Comment:
Since this part has been touched, maybe we can also get rid of the useless extra spaces?